modules/openapi-generator/src/main/resources/dart/libraries/dio/serialization/json_serializable/build.yaml.mustache

```diff
@@ -0,0 +1,18 @@
+targets:
+  $default:
+    builders:
+      json_serializable:
+        options:
+          # Options configure how source code is generated for every
+          # `@JsonSerializable`-annotated class in the package.
+          #
+          # The default value for each is listed.
+          any_map: false
+          checked: true
+          create_factory: true
+          create_to_json: true
+          disallow_unrecognized_keys: true
+          explicit_to_json: true
+          field_rename: none
+          ignore_unannotated: false
+          include_if_null: false
```

Comment:
This should be `true`. Mandatory fields should be present in json if they are nullable.

Reply:
I've updated the property declarations to override `IncludeIfNull` for each property.